torch/_functorch/partitioners.py

```diff
@@ -824,8 +824,7 @@ def solve_min_cut(
             if node.op == "call_function" and hasattr(node.target, "_overloadpacket")
         }
         ops_ignored = joint_module_ops - {str(i) for i in op_types.recomputable_ops}
-        print("Ops banned from re-materialization: ", ops_ignored)
-        print()
+        logger.info("Ops banned from re-materialization: ", ops_ignored)
```

Comment:
```suggestion
        logger.info("Ops banned from re-materialization: %s", ops_ignored)
```
Unlike print, second args will be ignored otherwise